File: Projekt2_java/src/projekt2/zwierzeta/Owca.java
package projekt2.zwierzeta;

import projekt2.*;

import java.awt.*;

public class Owca extends Zwierze {
    public Owca(Pair pole, Swiat swiat) {
        super(4, 4, 0, pole, swiat);
    }

    @Override
    public Organizm potomek(Pair pole) {
        return new Owca(pole, swiat);
    }

    @Override
    public Color rysuj() {
        return Color.LIGHT_GRAY;
    }

    @Override
    public String getNazwa() {
        return "owca";
    }
}

File: Projekt1_cpp/Projekt1_cpp/Owca.cpp
#include "Owca.h"

Owca::Owca(pair<int, int> pole, Swiat* swiat)
	:Zwierze(4, 4, 0, pole, swiat) {}

Organizm* Owca::potomek(pair<int, int>pole) {
	return new Owca(pole, swiat);
}

void Owca::rysowanie() {
	cout << 'o';
}

string Owca::getNazwa() {
	return "owca";
}